extensionsGallery.json

```diff
@@ -63,6 +63,59 @@
 					"statistics": [],
 					"flags": "preview"
 				},
+				{
+					"extensionId": "10",
+					"extensionName": "vscodeintellicode",
+					"displayName": "Visual Studio IntelliCode",
+					"shortDescription": "AI-assisted development",
+					"publisher": {
+						"displayName": "Microsoft",
+						"publisherId": "Microsoft",
+						"publisherName": "Microsoft"
+					},
+					"versions": [
+						{
+							"version": "1.2.1",
+							"lastUpdated": "10/29/2019",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
+									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/agent/agent-0.43.0.vsix"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Icons.Default",
+									"source": "https://intellicodeextensions.blob.core.windows.net/intellicodeextensions/extensions/azuredatastudio/1.2.1/icon.png"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Content.Details",
+									"source": "https://intellicodeextensions.blob.core.windows.net/intellicodeextensions/extensions/azuredatastudio/1.2.1/README.md"
+								}
```

Comment:
Do I need to upload the manifest (package.json) and License as well?

Reply:
It would be good if you did. Is the extension not open source? I don't see anything in the github repo

Reply:
it's not open source

Reply:
I'd recommend to have a GITHUB repo for your extension to host the license, manifest and icons...

Reply:
and also for users to open issues